controllers/restore_test.go

```diff
@@ -25,43 +32,51 @@ var _ = Describe("Restore", func() {
 					Namespace: "",
 				},
 				Status: v1beta1.RestoreStatus{
-					Phase: "INPROGRESS",
+					VeleroRestore: &veleroapi.Restore{
+						Status: veleroapi.RestoreStatus{
+							Phase: veleroapi.RestorePhaseInProgress,
+						},
+					},
 				},
 			}
 
 			Expect(isRestoreFinsihed(&rhacmRestore)).Should(BeFalse())
 
-			rhacmRestore.Status.Phase = "Completed"
+			setVeleroPhase(&rhacmRestore, veleroapi.RestorePhaseCompleted)
 			Expect(isRestoreFinsihed(&rhacmRestore)).Should(BeTrue())
-			rhacmRestore.Status.Phase = "Failed"
+			setVeleroPhase(&rhacmRestore, veleroapi.RestorePhaseFailed)
 			Expect(isRestoreFinsihed(&rhacmRestore)).Should(BeTrue())
-			rhacmRestore.Status.Phase = "PartiallyFailed"
+			setVeleroPhase(&rhacmRestore, veleroapi.RestorePhasePartiallyFailed)
 			Expect(isRestoreFinsihed(&rhacmRestore)).Should(BeTrue())
-			rhacmRestore.Status.Phase = "FailedValidation"
+			setVeleroPhase(&rhacmRestore, veleroapi.RestorePhaseFailedValidation)
 			Expect(isRestoreFinsihed(&rhacmRestore)).Should(BeTrue())
 
-			rhacmRestore.Status.Phase = "InvalidStatus"
+			Expect(isRestoreFinsihed(nil)).Should(BeFalse())
+
+			rhacmRestore.Status.VeleroRestore = nil
 			Expect(isRestoreFinsihed(&rhacmRestore)).Should(BeFalse())
-		})
 
-		It("getVeleroRestoreName should return the expected name", func() {
-			rhacmRestore := v1beta1.Restore{
-				TypeMeta: metav1.TypeMeta{
-					APIVersion: "cluster.open-cluster-management.io/v1beta1",
-					Kind:       "Restore",
-				},
-				ObjectMeta: metav1.ObjectMeta{
-					Name:      restoreName,
-					Namespace: "",
-				},
-				Spec: v1beta1.RestoreSpec{
-					BackupName: "the-latest-backup",
-				},
-			}
+		})
+		/*
```

Comment:
I'm never a big fan of commenting out code. If you don't need it, just remove it, you can go back to the previous commit if you need to bring it back, unless you intend to use it later. 😄

Reply:
Right! It's a cruft: see for example [this](https://github.com/identitatem/dex-operator/pull/18#discussion_r690918118) .